.github/workflows/ci.yml

```diff
@@ -146,10 +146,10 @@ jobs:
           if: steps.cache-source-deps.outputs.cache-hit != 'true' && contains(matrix.os, 'windows')
           shell: bash -l {0}
           run: |
-            # Compilation related dependencies 
+            # Compilation related dependencies
             mamba install cmake compilers make ninja pkg-config
             # Actual dependencies
-            mamba install -c robotology yarp wearables idyntree matio-cpp yarp-telemetry
+            mamba install -c robotology yarp wearables idyntree matio-cpp
```

Comment:
Not related to this PR, but `wearables` should not be here

Reply:
I did it here https://github.com/robotology/wearables/pull/155/commits/7190c9ed7848f311d6b26474ba644a99e4f9a30f